animation_nodes/nodes/number/offset_number.pyx

```diff
@@ -0,0 +1,16 @@
+from ... data_structures cimport Falloff, FalloffEvaluator, FloatList, VirtualDoubleList, Vector3DList
```

Comment:
I think this should be a `c_utils` file in the `/gpencil` directory. We can move it later if needed.

Reply:
Okay.